### DISCUSSION TITLE
How to use badges with special characters?

### DISCUSSION THREAD

**Discussion by kermanx** [badges/shields#7580] [Q&A] [answered]
I want to use a shield badge like this: `https://img.shields.io/badge/-_Kerman-00a1d6?style=flat-square&logo=bilibili&logoColor=white`

But it outputs like this:
<img src="https://img.shields.io/badge/-@_Kerman-00a1d6?style=flat-square&logo=bilibili&logoColor=white"/></a>

As you can see, `_` character is disappeared.

How to use `_` in a badge?

Waiting for help...🙏

**Comment by calebcartwright** [comment, marked as answer]
Use double underscores:

https://img.shields.io/badge/-@__Kerman-00a1d6?style=flat-square&logo=bilibili&logoColor=white
![](https://img.shields.io/badge/-@__Kerman-00a1d6?style=flat-square&logo=bilibili&logoColor=white)

https://shields.io/#your-badge

**Reply by kermanx** [reply]
Wow. Thanks a lot!

**Reply by calebcartwright** [reply]
yw!
